<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student</class>
 <widget class="QWidget" name="Student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>周一机房余量</string>
   </property>
   <widget class="QTableWidget" name="tableWidget_roomMargin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>机房ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>机房号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>上午余量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>下午余量</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>预约日期</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>预约时间</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>预约机房</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_orderDate">
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>周一</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>周二</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>周三</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>周四</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>周五</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_orderTime">
      <item>
       <property name="text">
        <string>上午</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>下午</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_orderRoom">
      <item>
       <property name="text">
        <string>1号机房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2号机房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3号机房</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_applyOrder_checkMargin">
      <property name="text">
       <string>查看机房余量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_applyOrder">
      <property name="text">
       <string>申请预约</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>641</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_checkSelfOrder">
      <property name="text">
       <string>查看自己预约</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_checkAllOrder">
      <property name="text">
       <string>查看所有预约</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancleOrder">
      <property name="text">
       <string>取消预约预约</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
